### PR TITLE
fix(tabs): invert scroll functions when rtl, flip icons

### LINKED ISSUE #1304: [bug] `<rh-tabs>` mobile overflow buttons are broken for RTL
mobile overflow button are broken for RTL

icons face the wrong way and do scroll the overlfow

image

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

1. Added support for rtl overflow buttons, inverting scroll methods and icons

Closes #1304

## Testing Instructions

1. View [tabs RTL demo](https://deploy-preview-1575--red-hat-design-system.netlify.app/elements/tabs/demo/right-to-left/) 

## Notes to Reviewers

View at a screen size where tabs overflows and overflow buttons appear

### CODE REVIEW

File: .changeset/curly-jobs-eat.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": patch
+---
+
+`<rh-tabs>`: added support for rtl language overflow scroll buttons'
```
Comment (bennypowers) [inline .changeset/curly-jobs-eat.md:5]:
```suggestion
`<rh-tabs>`: added support for rtl language overflow scroll buttons
```

File: elements/rh-tabs/rh-tabs.ts [hunk 2]
```diff
@@ -192,14 +196,15 @@ export class RhTabs extends LitElement {
...
     return html`
-      <div id="rhds-container" class="${classMap({ [on]: !!on })}">
+      <div id="rhds-container" class="${classMap({ [on]: !!on, rtl })}">
         <div part="container" class="${classMap({ overflow: this.#overflow.showScrollButtons })}">
           <div part="tabs-container">${!this.#overflow.showScrollButtons ? '' : html`
             <button id="previousTab" tabindex="-1"
                     aria-label="${this.getAttribute('label-scroll-left') ?? 'Scroll left'}"
                     ?disabled="${!this.#overflow.overflowLeft}"
-                    @click="${() => this.#overflow.scrollLeft()}">
+                    @click="${() => !rtl ? this.#overflow.scrollLeft() : this.#overflow.scrollRight()}">
```
Comment (bennypowers) [inline elements/rh-tabs/rh-tabs.ts:207]: test this...

**Review by bennypowers — CHANGES_REQUESTED** [review]

File: .changeset/curly-jobs-eat.md (hunk 1 above)
Comment (zeroedin) [reply .changeset/curly-jobs-eat.md:5]: ooh good typo catch

File: elements/rh-tabs/rh-tabs.ts (hunk 2 above)
Comment (zeroedin) [reply elements/rh-tabs/rh-tabs.ts:207]: @bennypowers PTAL, definitely need a better way to add `dir="rtl"` to the test so im not having to clone and reconnect the component to pick up the ancestor change.

Comment (bennypowers) [reply elements/rh-tabs/rh-tabs.ts:207]: i think you can just call connectedCallback again

https://github.com/RedHat-UX/red-hat-design-system/blob/b443fe21700a2a514f9aa24bc291be34e8bd370c/lib/DirController.ts#L16

Comment (zeroedin) [reply elements/rh-tabs/rh-tabs.ts:207]: 🥇

Comment (zeroedin) [reply elements/rh-tabs/rh-tabs.ts:207]: why i just glossed over that such obvious fix /smh 🙃

**Review by bennypowers — APPROVED** [review]

**Pull request merged**